src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -289,11 +289,17 @@ export class SavedObjectsRepository {
 
     let savedObjectWorkspaces = workspaces;
```

Comment:
Will the savedObjectWorkspaces be set to undefined if `options.workspaces` not exists?

Reply:
Good catch, for such case, we need to call get when create-with-override